docs/hw/gpu.rst

```diff
@@ -698,6 +698,7 @@ unk?  \???? 0x132 GP102  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?
 unk?  \???? 0x134 GP104  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?     ?      ?   ?   ?   ?
 unk?  \???? 0x136 GP106  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?     ?      ?   ?   ?   ?
 unk?  \???? 0x137 GP107  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?     ?      ?   ?   ?   ?
+unk?  \???? 0x138 GP108  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?     ?      ?   ?   ?   ?
```

Comment:
Add the pci id here as well.  If this GPU has a HDA device, fill it in; otherwise, write \- here.